libs/SalesforceAnalytics/SalesforceAnalytics/Supporting Files/SalesforceAnalytics-Prefix.pch

```diff
@@ -4,4 +4,5 @@
 
 #ifdef __OBJC__
     #import <Foundation/Foundation.h>
+    #import "SFSDKAnalyticsLogger.h"
```

Comment:
The cocoapods won't compile because their pch will be missing that line.
To add that import to the pod's pch, we need to do [that](https://stackoverflow.com/questions/21577655/how-can-i-include-a-pch-for-a-specific-pod-in-cocoapods)